example6.js

```diff
@@ -36,14 +41,37 @@ new Cam({
 
     // Use Nimbe's flow to execute ONVIF commands in sequence
     flow.series([
```

Comment:
I think we can use promises here since engine version in package.json is ` "node": ">=6.0"`

Reply:
Good idea. All my other examples use nimble so I will have a go with promises on all of them. Could we merge without example6 and leave the example in my branch?